pset-functions.js

```diff
@@ -76,6 +87,32 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const scrabbleScore = word => {
+    const letters = word.toUpperCase().split('')
+    let sum = 0;
+
+    letters.forEach(function(letter){
```

Comment:
Sammi, I love that you considered possible edge cases and chained methods on line 91. Great use of the forEach method to iterate as well; and way to use the built in array method .includes! This is very well written code.